puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_plugin.sh

```diff
@@ -46,7 +46,7 @@ fi
 ) > $APP_ROOT/config/database.yml
 
 # Create DB first in development as migrate behaviour can change
-bundle exec rake db:drop db:create db:migrate
+bundle exec rake db:drop db:create db:migrate DISABLE_DATABASE_ENVIRONMENT_CHECK=true
```

Comment:
Give we configure the development database above, shouldn't we just use ```RAILS_ENV=development``` instead?

Reply:
Let's give it a try